projects/ngx-meta/schematics/ng-add/testing/should-add-root-provider.ts

```diff
@@ -17,7 +17,7 @@ export const shouldAddRootProvider = (
       standalone,
     )
     expect(appConfigOrAppModuleContents).toContain(
-      `import { ${providerTestCase.symbol} } from '${LIB_NAME}/${providerTestCase.entrypoint}`,
```

Comment:
⚠️ Important! Otherwise the test case about `open-graph-profile` was not correct. Given at first, it was importing from `open-graph-profile` but expected to import from `open-graph`. Though given the final `'` was missing, and import was `open-graph-profile`, it was actually importing from something starting with `import ...  from '.../open-graph` soooo it passed when it shouldn't.